plugins/modules/helm_plugin_info.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: helm_plugin_info
+short_description: Gather information about Helm plugins
+version_added: "0.11.0"
+author:
+  - Abhijeet Kasurde (@Akasurde)
+requirements:
+  - "helm (https://github.com/helm/helm/releases)"
+description:
+  -  Gather information about Helm plugins installed in namespace.
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  release_namespace:
+    description:
+      - Kubernetes namespace where the helm plugins are installed.
+    required: true
+    type: str
+    aliases: [ namespace ]
+
+#Helm options
```

Comment:
Not a blocker at all for this PR but I feel like we should break these common options into doc fragments etc to cut down on the repetition

Reply:
I will make a separate PR for this. Thanks for the suggestions.